CHANGELOG.rst

```diff
@@ -15,6 +15,8 @@ Added
 - Added ``special_available_tags`` which stores `"untagged"` and `"any"` if they can be used from an Interface.
 - Added ``maxsize_multiplier`` on ``event_buffer_conf``, which will multiply the ``maxsize`` value of the queue. By default, all KytosEventBuffer who use a bounded queue will have ``maxsize_multiplier: 2``. This default is reasonable to work out of the box with kytos-ng core NApps. But, if you have other NApps who tend to produce too many events you might want to either increase the size of the queue with and/or increase the number of max workers in the thread pool if the event handler is running on a thread pool. Typically, you'll want to first start adjusting the number of workers in the thread pool.
 - Introduced a new ``meta`` on ``KytosBuffers``, which is meant for general core control events.
+- Added ``api`` threadpool. This is used by the ASGI server to handle requests to non async endpoints.
```

Comment:
I wonder if we should also further explain in a case of an upgrade to either regenerate kytos.conf, and/or indeed just set this, otherwise they'll end up with `40`, which already solves part of the problem, but let's further clarify here. Maybe this is also already clear an we don't need more info. Let me know what you think.

Reply:
Maybe we should. The thing is that in the past, when we changed the configuration files, we never mentioned in the change log to regenerate them.

Reply:
Indeed, and this might get repetitive too. What if we just added just a general info like:

```
General Information
===================

- kytos.conf.template has changed, you might want to regenerate it if you want to set non default values.
```


This coupled with the specific one that you've added would make crystal clear that if network operators don't regenerate or set they values, then they want the default ones.Let me know if you have any other suggestion to simplify or make it clear.

Reply:
Added in the note to the changelog, good idea, and good pattern for future changelogs.